module/solutions/helper.js

```diff
@@ -1566,44 +1617,56 @@ module.exports = class SolutionsHelper {
    * @returns {Object} - Details of the solution.
    */
 
-  static verifyLink(link = "", bodyData = {}, userId = "", userToken = "", createProject = true ) {
+  static verifyLink(
+    link = "",
+    bodyData = {},
+    userId = "",
+    userToken = "",
+    createProject = true
+  ) {
     return new Promise(async (resolve, reject) => {
       try {
-       
         let verifySolution = await this.verifySolutionDetails(
           link,
           userId,
           userToken
         );
-        
+
         let checkForTargetedSolution = await this.checkForTargetedSolution(
           link,
           bodyData,
           userId,
           userToken
         );
-        if( !checkForTargetedSolution || Object.keys(checkForTargetedSolution.result).length <= 0 ) {
+        if (
+          !checkForTargetedSolution ||
+          Object.keys(checkForTargetedSolution.result).length <= 0
+        ) {
           return resolve(checkForTargetedSolution);
         }
-        
+
         let solutionData = checkForTargetedSolution.result;
 
-        if( solutionData.type == constants.common.OBSERVATION ) {
+        let solutionStatus = solutionData.status === "inactive" ? false : true;
```

Comment:
@ankitpws We should use constants as constants.common.INACTIVE also the variable name can is  isSolutionActive ?

Reply:
used constants common variable and changed isSolutionActive